### PR TITLE
Install bind source dependencies properly

### PR DESCRIPTION
**Pull request by dzedro**
- Verification run: 
[12 SP1](http://10.100.12.155/tests/6208#step/bind/73)
[12 SP2](http://10.100.12.155/tests/6197#step/bind/73)
[12 SP3](http://10.100.12.155/tests/6199#step/bind/73)
[15](http://10.100.12.155/tests/6203#step/bind/64)

### CODE REVIEW

File: tests/console/bind.pm [hunk 1]
```diff
@@ -17,25 +17,74 @@ use strict;
...
+fi
+
+function finish {
+ rm -f /tmp/SUSEConnect.txt
+}
+trap finish EXIT
+
+SUSEConnect -s >$SUSECONNECT
+
+# remove added products if /tmp/REMOVE_ADDED_PRODUCTS ecists otherwise add them
```
Comment (rwx788) [inline tests/console/bind.pm]: s/ecists/exists

**Comment by dzedro** [comment]
hmm we still have ninja keys ? :) I can download it

**Comment by rwx788** [comment]
@dzedro Bug is still open: https://bugzilla.suse.com/show_bug.cgi?id=1041747
For hyper-V it works if type really-really slow.

**Comment by dzedro** [comment]
Script will be downloaded, even when it will most probably not run on hyper-V or ppc64le.

**Pull request merged**